src/App.js

```diff
@@ -4,36 +4,91 @@ import Experience from "./Components/Experience";
 import Avatar from "./Components/Avatar";
 import Footer from "./Components/Footer";
 import "./App.css";
-
+import { useState } from "react";
 
 function App() {
-  const HEADING_TEXT = "My Portfolio";
-  const SIGNUP_BUTTON_TEXT = "Sign up";
+  const [isEditMode, setIsEditMode] = useState(false);
+
+  const [headerHeading, setHeaderHeading] = useState("My Portfolio");
+  const toggleEditMode = () => {
+    setIsEditMode(!isEditMode);
+  };
+  const Edit_BUTTON_TEXT = isEditMode ? "Save" : "Edit";
```

Comment:
const HEADER_BUTTON_TEXT = isEditMode ? "Save" : "Edit"; I suggest to update the text constant to something like this. EDIT_BUTTON_TEXT suggest that this button can only be used for editing but we implement a save operation on it as well

Reply:
fixed